setup.cfg

```diff
@@ -12,6 +12,6 @@ parentdir_prefix = Flocker-
 
 [flake8]
 exclude=flocker/_version.py,flocker/restapi/docs/hidden_code_block.py
-# Ignore defaults + E731
+# Ignore defaults + E731, E305
 # http://pep8.readthedocs.io/en/latest/intro.html#error-codes
-ignore=E123,E226,E241,E731,W503
+ignore=E123,E226,E241,E731,E305,W503
```

Comment:
305 is "expected 2 blank lines after end of function or class"

Is there a reason we dont want this anymore? I'm indifferent, just curious.